src/EA.Weee.Web/Areas/Producer/Views/Producer/OrganisationDetails.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@{
+    ViewBag.Title = "View organisation - GOV.UK";
+}
+
+@section full_main_content
+{
+    <header class="hgroup">
+        <h1 class="govuk-heading-l">View organisation</h1>
```

Comment:
Are we not splitting the title to be separate from the "- GOV.UK" like on other views with this format
 
const string title = "...";
ViewBag.Title = title + " - GOV.UK";

Then using @title in the heading

Reply:
Updated...